versioned_docs/version-v4.0.0/node/backup-restore-update.md

```
@@ -0,0 +1,464 @@
+
+2. **Stop the hoprd Services**
+
+    Navigate to the `compose` folder and stop the `hoprd` services by running the following command:
+    ```
+    COMPOSE_PROFILES=hoprd docker compose down
+    ```
+
+3. **Restore the Identity File**
+
+    Inside the `compose` folder, copy your `hopr.id` file into:
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)